tools/Makefile

```diff
@@ -98,7 +98,7 @@ openfst-$(OPENFST_VERSION).tar.gz:
 		cp -p "$(DOWNLOAD_DIR)/openfst-$(OPENFST_VERSION).tar.gz" .; \
 	else \
 		$(WGET) -T 10 -t 1 http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-$(OPENFST_VERSION).tar.gz || \
-		$(WGET) -T 10 -t 3 https://www.openslr.org/resources/2/openfst-$(OPENFST_VERSION).tar.gz; \
+		$(WGET) -T 10 -t 3 -O openfst-$(OPENFST_VERSION).tar.gz https://www.openslr.org/resources/2/openfst-$(OPENFST_VERSION).tar.gz; \
```

Comment:
Hm. I think an easier and better fix would be to add the --continue flag.  You can add it to all of those wget commands.

Reply:
It's a better idea, and i change those case math regex `\|\|\s*\\*\s*(wget|\$\(WGET\))`. I think it's all of that.